docker/gatling/Dockerfile

```diff
@@ -1,17 +1,15 @@
 # Dockerfile for building gatling image with benchmark Tests
 FROM denvazh/gatling:2.3.1
 
-# TODO - Discuss if this is needed now.
-# ADD http://abondance.internal.forgerock.com/pkg/clients/gatling/ext/ldap/gatling-ldap-full-0.0.4-SNAPSHOT.zip /
-# RUN mkdir /ldap-plugin
-# RUN unzip /gatling-ldap-full-0.0.4-SNAPSHOT.zip -d /ldap-plugin
-
-#RUN cp /ldap-plugin/* /opt/gatling/lib/
-# Add local files
+# In case of you want to add custom gatling libraries
+# make sure you you add them to image and copy to apropriate place
+# RUN cp /ldap-plugin/* /opt/gatling/lib/
 
+# Using custom gatling startup scripts for easier passing of variables
+# through helm charts
 COPY custom-gatling.sh /custom-gatling.sh
 RUN chmod +x /custom-gatling.sh
-COPY simulations /opt/gatling/user-files/
+COPY simulations /opt/gatling/user-files/simulations
```

Comment:
Perhaps we should look at loading the simulations via a configmap mounted as a volume?
Helm can slurp up files in a folder and stick them in a configmap

Reply:
I agree that will be better for developing and testing benchmarks (no need for rebuilding docker image). I will merge this change and will add this in following set of changes.